medical_pathology/README.rst

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+======================
+Odoo Medical Pathology
+======================
+
+This module extends Odoo Medical with functionality for pathologies (diseases).
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/159/9.0
+
+Known issues / Roadmap
+======================
+
```

Comment:
Please add a note here that this module replaces the pathology logic from `medical_disease`. Please also note that the remaining `medical_disease` logic is in `medical_patient_disease`. 

This will help anyone upgrading from v8, as well as anyone creating OpenUpgrade logic for this repo.